src/client.ts

```diff
@@ -80,12 +83,7 @@ export class SubsquidClient {
   async query (
     input: string
```

Comment:
The type here says `string`, but is this not the method that should take a structured query object and return the derived result type? Or am I reading it wrong?